lib/kitchen/provisioner/chef_base.rb

```diff
@@ -570,7 +577,7 @@ def script_for_omnibus_version
           config[:require_chef_omnibus], powershell_shell?, install_options
         )
         config[:chef_omnibus_root] = installer.root
```

Comment:
Do you know offhand where else `:chef_omnibus_root` might be needed? I see it's set here and in `script_for_product`.  I've only seen it checked in `install_options` - but we set `install_options` above,  before we set the `config` value that it references.

Reply:
I see it read in some of the other provisioners to get things like the `chef-apply` or `chef-solo` path